src/assets/wise5/authoringTool/node/node-authoring/node-authoring.component.ts

```diff
@@ -292,37 +292,56 @@ export class NodeAuthoringComponent implements OnInit {
     this.hideComponentAuthoring();
   }
 
-  protected deleteButtonClicked(): void {
+  protected deleteComponents(): void {
     this.scrollToTopOfPage();
     this.hideComponentAuthoring();
-
-    let confirmMessage = '';
     const selectedComponentNumbersAndTypes = this.getSelectedComponentNumbersAndTypes();
```

Comment:
Since we're not using ```selectedComponentNumbersAndTypes``` anywhere else, we don't need to declare it and just pass in the result along to confirmDeleteComponent()?
```suggestion
    if (this.confirmDeleteComponent(this.getSelectedComponentNumbersAndTypes())) {
```